Comments/ClientApp/src/components/LeadInfo/LeadInfo.js

```diff
@@ -0,0 +1,65 @@
+﻿import React, { Fragment } from "react";
+import { withRouter } from "react-router-dom";
+import Helmet from "react-helmet";
+import { withHistory } from "../HistoryContext/HistoryContext";
+import { Breadcrumbs } from "../Breadcrumbs/Breadcrumbs";
+import { Header } from "../Header/Header";
+
+export const LeadInfo = () => {
+	const emailAddress = "DITApplicationSupport@nice.org.uk"
+	const emailSubject = encodeURIComponent("New commenting lead request");
+	const emailBody = encodeURIComponent("Dear Application Support\r\rMy name is:\rI want to be a commenting lead for the following organisation:\r\rKind regards");
+
+	return (
+		<Fragment>
+			<Helmet>
+				<title>What is a commenting lead?</title>
+			</Helmet>
+			<main role="main">
+				<div className="container">
+					<div className="grid">
+						<div data-g="12">
+							<Breadcrumbs links={[{label:"Home",url:"/"},{label:"All consultations", url:"/guidance/inconsultation"}]} />
+
+							<Header title="What is a commenting lead?" />
+							<p>Commenting leads are representatives of an organisation who collate feedback on NICE consultations from colleagues within their organisation. They are also responsible for submitting their organisation's consultation response to NICE.</p>
+							<p>Leads are now able to do this online using our commenting system, but will require a set of permissions which can be assigned on request.</p>
+						
+							<div className="grid">
+								<div data-g="8">
+									<h2>How does it work?</h2>
+									<p>
+										Once you have signed up to be the lead for a consultation, you will be able to:
+										<ul>
+											<li>share the online consultation with colleagues from your organisation</li>
+											<li>review and edit their comments</li>
+											<li>combine the collated comments into a single response</li>
+											<li>submit your organisation's final response.</li>
+										</ul>
+										You can be the lead for multiple consultations.
+									</p>
```

Comment:
```suggestion
									<p>
										Once you have signed up to be the lead for a consultation, you will be able to:</p>
										<ul>
											<li>share the online consultation with colleagues from your organisation</li>
											<li>review and edit their comments</li>
											<li>combine the collated comments into a single response</li>
											<li>submit your organisation's final response.</li>
										</ul>
										<p>You can be the lead for multiple consultations.
									</p>
```
React seems to be complaining about the list being nested in a paragraph. I've changed it above, but the formatting/indentation might be off.